Haxe/Scripts/cases/TestTArray.hx

```diff
@@ -361,25 +361,39 @@ class TestTArray extends buddy.BuddySuite {
         for (s in [obj.set, ReflectAPI.callMethod(obj, "ufuncGet_set", []), obj2.set, ReflectAPI.callMethod(obj2, "ufuncGet_set", []), TSet.create(new TypeParam<FString>())]) {
           s.Contains("Test").should.be(false);
           s.Contains("Test2").should.be(false);
+          s.Num().should.be(0);
+          s.IsValidId(FSetElementId.FromInteger(-1)).should.be(false);
+          s.IsValidId(FSetElementId.FromInteger(0)).should.be(false);
           var t1 = s.Add("Test");
           s.Contains("Test").should.be(true);
-          s.Contains("Test2").should.be(false);
+          s.Contains("Test2").should.be(false);         
+          s.Num().should.be(1);
+          s.IsValidId(FSetElementId.FromInteger(0)).should.be(true);
+          [for( item in s) item.toString()].should.containAll(["Test"]);         
           var scpy = s.copy();
           var t2 = s.Add("Test2");
           s.Contains("Test").should.be(true);
           s.Contains("Test2").should.be(true);
+          s.Num().should.be(2);
+          [for( item in s) item.toString()].should.containAll(["Test", "Test2"]);      
           s.Remove(t1);
           s.Contains("Test").should.be(false);
           s.Contains("Test2").should.be(true);
+          s.Num().should.be(1);
+          s.IsValidId(FSetElementId.FromInteger(0)).should.be(false);
```

Comment:
I think we can't assume that. It could break if the underlying TSet implementation changes. If you still want to test this, I recommend getting the Id of `Test2` and ensuring that both ThisId + 1 and ThisId - 1 are not valid